paket.dependencies

```diff
@@ -15,33 +15,33 @@ source https://api.nuget.org/v3/index.json
 // nuget Microsoft.AspNetCore.All 2.0.0
 
 // DOTNETCORE & PAKET ISSUES
-nuget Microsoft.NETCore.App 2.2.0
-nuget Microsoft.NETCore.DotNetAppHost 2.2.0
-nuget Microsoft.NETCore.DotNetHostResolver 2.2.0
-nuget Microsoft.NETCore.DotNetHostPolicy 2.2.0
+nuget Microsoft.NETCore.App 2.2.4
+nuget Microsoft.NETCore.DotNetAppHost 2.2.4
+nuget Microsoft.NETCore.DotNetHostResolver 2.2.4
+nuget Microsoft.NETCore.DotNetHostPolicy 2.2.4
 nuget NETStandard.Library 2.0.3
 
 // PRODUCTION STUFF
 nuget Microsoft.Extensions.Configuration 2.2.0
```

Comment:
You can remove these Configuration ones, moved to https://github.com/Informatievlaanderen/api/blob/master/src/Be.Vlaanderen.Basisregisters.Api/paket.template#L54-L60